controllers/hcpvaultsecretsapp_controller.go

```diff
@@ -122,6 +122,9 @@ func (r *HCPVaultSecretsAppReconciler) Reconcile(ctx context.Context, req ctrl.R
 	params := &hvsclient.OpenAppSecretsParams{
 		Context: ctx,
 		AppName: o.Spec.AppName,
+		Types: []string{
+			helpers.HVSSecretTypeKV,
+		},
```

Comment:
Since multiple secret types can come back in the open app, this is requesting only "kv" secrets for now.